Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -6713,25 +7158,90 @@ script:
     - contextPath: Panorama.Content.Download.Details
       description: The job ID details.
       type: String
+    polling: true
   - arguments:
     - description: Serial number of the firewall on which to run the command. Use only for a Panorama instance.
       name: target
-    description: Installs the latest content update.
+    - description: Job ID for a running download process. Used for status polling.
+      name: job_id
+      hidden: true
+    description: Installs the latest app/threat dynamic update.
     name: pan-os-install-latest-content-update
     outputs:
     - contextPath: Panorama.Content.Install.JobID
       description: The job ID of the installation.
       type: String
-    - contextPath: Content.Install.Status
```

Comment:
@rshunim We should be able to ignore the pre-commit warning about this change.  While working on these enhancements I found what appears to have been a typo in the YML file, and this should have been `Panorama.Content.Install.Status` all along. The integration python code did not output the job status to a different context prefix than the job id.